### PR TITLE
Add Component-Types

### PR DESCRIPTION
**Pull request by hkhattabii**

### CODE REVIEW

File: src/paths/componentTypes/fetchByName.yaml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  '200':
+    description: Returns a component type regarding to the name passed in the path
+    content:
+      application/json:
+        schema:
+          $ref: "../../schemas/componentTypes/ComponentTypeDto.yaml"
+  "401":
+    $ref: "../../responses/TokenResponse.yaml"
+  "403":
+    $ref: "../../responses/components/ResponseComponentForbidden.yaml"
```
Comment (BerrevoetsRobbe) [inline src/paths/componentTypes/fetchByName.yaml:21]: normally these calls are not authorized calls, so they can't return 401/403?

**Review by BerrevoetsRobbe — COMMENTED** [review]
questions

File: src/paths/componentTypes/fetchByName.yaml (hunk 1 above)
Comment (hkhattabii) [reply src/paths/componentTypes/fetchByName.yaml:21]: yes, 
- when there is no token  : The api returns 403 Forbidden.
- When there is wron token ( e.g. format ) : The api returns 401

Comment (BerrevoetsRobbe) [reply src/paths/componentTypes/fetchByName.yaml:21]: okay, I see. according to our implementation at io.metadew.iesi.server.rest.configuration.security.jwt.JwtWebSecurityConfiguration#configure(org.springframework.security.config.annotation.web.builders.HttpSecurity) the user should be authenticated before calling any endpoint (including action-types, component-types and connection-types)

**Review by BerrevoetsRobbe — APPROVED** [review]

**Review by BerrevoetsRobbe — APPROVED** [review]

**Pull request merged**